packages/create-web-scripts-library/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "@spotify/web-scripts/config/tsconfig.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "incremental": true,
+    "isolatedModules": false,
+    "noEmit": true,
+    "declaration": false,
```

Comment:
for both noEmit and declaration, doesn't web-scripts handle these flags? I don't think we should imply they need to be managed by repeating them here.

Reply:
huh, I have no clue why I have these differences like this. Will clear them out.